examples/api/test/material/progress_indicator/linear_progress_indicator.0.dart

```diff
@@ -0,0 +1,24 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Shouldn't the name of this file be: `examples/api/test/material/progress_indicator/linear_progress_indicator.0_test.dart`?

Reply:
Fixed :)